x-pack/test_utils/jest/config.contract.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { RESERVED_DIR_JEST_CONTRACT_TESTS } from '../../../src/dev/constants';
+import config from './config';
+
+export default {
+  ...config,
+  testMatch: [
+    `**/${RESERVED_DIR_JEST_CONTRACT_TESTS}/**/*.test.{js,ts,tsx}`,
```

Comment:
maybe we should look for `*.contract.test.{js,ts,tsx}`
This way it would always run as unit tests along with existing unit tests.
We could then hard-code env vars here to trigger the tooling to run as online?

After that we could add the commands to run them inside `test/scripts/jenkins_xpack.sh`
That way it will run on CI. Thoughts on this plan @mistic @spalger @nchaulet?

Reply:
`contract.test.*` seems fine for me, and yes they should be able to run as unit tests if there is no flag online passed.